geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -798,6 +798,13 @@ const MainController = function(
    */
   this['selectedLayers'] = [];
 
+  /**
+   * @type {function}
+   */
+  this['selectedLayersLength'] = function() {
```

Comment:
Brackets were used for closure-compiler. It is not necessary anymore. Better not to use them in new code.

Reply:
I wondered but keep brackets for coherence with the rest of the file, I can change if needed.